sources/packages/backend/libs/sims-db/src/entities/student-loan-balances.model.ts

```diff
@@ -0,0 +1,42 @@
+import { Column, Entity, PrimaryGeneratedColumn } from "typeorm";
+import { TableNames } from "../constant";
+import { RecordDataModel } from "./record.model";
+import { numericTransformer } from "../transformers/numeric.transformer";
+
+/**
+ * Stores files information and content from NSLSC
+ * Loan Balance file.
+ */
+@Entity({ name: TableNames.StudentLoanBalances })
+export class StudentLoanBalances extends RecordDataModel {
```

Comment:
As a general pattern I can observe, the entity modal class names are singular. @andrewsignori-aot @andrepestana-aot @sh16011993 @lewischen-aot  thoughts??

Reply:
Yes, table names are plural and entity names singular.